modulefiles/orion

```diff
@@ -0,0 +1,34 @@
+#%Module######################################################################
```

Comment:
@aerorahul I changed modulefile on orion and hera in lua format. Would you remove orion and hera?

Reply:
Noted. Done.